libcodechecker/logger.conf

```diff
@@ -0,0 +1,56 @@
+{
+  "version": 1,
+  "disable_existing_loggers": true,
+  "formatters": {
+    "brief": {
+      "format": "[%(asctime)s] - %(message)s",
+      "datefmt": "%Y-%m-%d %H:%M"
+    },
+    "precise": {
+      "format": "[%(levelname)s][%(asctime)s] {%(name)s} [%(process)d] <%(thread)d> - %(filename)s:%(lineno)d %(funcName)s() - %(message)s",
```

Comment:
Space between `levelname` and `asctime`: `[%(levelname)s] [%(asctime)s]`

Reply:
I did not wanted to put spaces there, the same is valid for the default built-in format.